File: add/codesnippet/CSharp/p-system.web.configurati_176_1.cs

   
        // Show the use of the AppSettings property
        // to get the application settings. 
        static void GetAppSettings()
        {

            // Get the appSettings key,value pairs collection.
            NameValueCollection appSettings =
                WebConfigurationManager.AppSettings
                as NameValueCollection;

            // Get the collection enumerator.
            IEnumerator appSettingsEnum =
                appSettings.GetEnumerator();

            // Loop through the collection and 
            // display the appSettings key, value pairs.
            int i = 0;
            Console.WriteLine("[Display appSettings]");
            while (appSettingsEnum.MoveNext())
            {
                string key = appSettings.AllKeys[i].ToString();
                Console.WriteLine("Key: {0} Value: {1}",
                key, appSettings[key]);
                i += 1;
            }

            Console.WriteLine();
        }

File: add/codesnippet/VisualBasic/p-system.web.configurati_176_1.vb
   ' Show the use of AppSettings property
   ' to get the application settings.
   Shared Sub GetAppSettings()
      
      ' Get the appSettings key,value pairs collection.
        Dim appSettings As NameValueCollection = _
        WebConfigurationManager.AppSettings
      
      ' Get the collection enumerator.
        Dim appSettingsEnum As IEnumerator = _
        appSettings.GetEnumerator()
      
      ' Loop through the collection and 
      ' display the appSettings key, value pairs.
      Dim i As Integer = 0
      Console.WriteLine("[Display appSettings]")
      While appSettingsEnum.MoveNext()
         Dim key As String = appSettings.AllKeys(i)
            Console.WriteLine("Key: {0} Value: {1}", _
            key, appSettings(key))
         i += 1
      End While
      
      Console.WriteLine()
   End Sub 'GetAppSettings
   
   